src/views/ProfileView.vue

```diff
@@ -154,6 +156,15 @@
         });
       }
 
+      if (zuordnung.befristung) {
+        tempSchulDaten.labelAndValues.push({
+          label: t('profile.befristung'),
+          value: new Date(zuordnung.befristung).toLocaleDateString('de-DE'),
```

Comment:
Man darf nicht einfach die zurückgegebene Befristung anzeigen. Die Befristung einer Zuordnung ist in UTC bei uns gespeichert.

Beispiel: Bei Benutzeranlage gebe Ich eine Befristung von 31.07.2025 ein. Das heisst in diesem Fall, dass im Backend die Befristung im Format 31-07-2025 22H gespeichert wird (UTC). Wenn Ich diese Befristung dann wieder in deutsche Zeit umwandele, dann wird das Timezone abhängig sein, also entweder Winterzeit oder Sommerzeit. 

Bei Winterzeit sind wir in Deutschland bei UTC + 1 , das heisst es ist okay, wenn wir direkt die Befristung displayen. Rückgabe wird dann 31.07.2025.

Bei Sommerzeit sind wir bei UTC + 2, das heisst, wenn wir in DE Zeit umwandeln mit .toLocaleDateString dann wird 1 Tag dazu kommen und 01.08.2025 angezeigt (Was auch richtig ist aber der User will ja 31.07.2025 sehen und nicht 01.08.2025).

Hier ein aus PersonCreationView.vue 

```
  // Converts the ISO UTC formatted Befristung to the german local format, also ISO.
  const translatedBefristung: ComputedRef<string> = computed(() => {
    const ISOFormattedDate: string | undefined = schuleZuordnungFromCreatedKontext.value?.befristung;

    if (!ISOFormattedDate) {
      return t('admin.befristung.unlimitedSuccessTemplate');
    }

    // Parse the UTC date
    const utcDate: Date = new Date(ISOFormattedDate);

    // Subtract one day. The reason to substract it here is because when the UTC time from the backend gets converted back to the local german date here, it shows the next day
    // It's logical since we send the date in the first place as "31-07-2024 22H" UTC TIME which is "01-08-2024" 00H of the next day in MESZ (summer german time)
    // but the user obviously doesn't want to know that.
    if (utcDate.getTimezoneOffset() >= -120) {
      // Check if the timezone offset is 2 hours (indicating MESZ)
      // Subtract one day if in summer time (MESZ)
      utcDate.setDate(utcDate.getDate() - 1);
    }
    const germanDate: string = utcDate.toLocaleDateString('de-DE');

    return germanDate;
  });
```